### PR TITLE
docs(fire): record the fire subsystem scope boundary and sibling-repo criteria

### LINKED ISSUE #813: FIRE-21 — Scope decision: NFDRS and the sibling-repository boundary
**Labels:** `design`, `discussion`, `fire-weather`
**Milestone:** `v2.8 — Fire weather diagnostics`

### Summary

Record the decision about where `climate_indices` stops, so that future
contributors and issue triage have a clear rule rather than relitigating it.

### Proposed boundary

The dividing line is not drought versus wildfire. It is **meteorological and
climatological indices** versus **operational fire-danger and fire-behavior
modeling**.

In scope for `climate_indices`:
- Indices computable from standard meteorological and reanalysis fields
- Deterministic, well-published algorithms with authoritative reference code
- Anything that fits the existing NumPy + xarray + CF metadata + Dask pattern

Out of scope:
- NFDRS Ignition Component, Spread Component, Energy Release Component and
  Burning Index, which NWCG defines in terms of live and dead fuel moisture
  and fuel models rather than weather alone
- Fuel model catalogs, live and dead fuel state, and operational calibration
- Fire-behavior simulation and rate-of-spread modeling
- FWI2025 next-generation reformulations, at least initially

### Criteria for creating `fire_weather_indices` as a sibling repository

Create it when two or more of the following become true:

1. An implementation requires fuel-model or fuel-state data structures
2. An implementation requires operational calibration against fire-occurrence
   records
3. The fire code exceeds roughly 30 percent of the package's source volume
4. Fire-specific dependencies would be forced on all `climate_indices` users
5. The release cadence needs to diverge

### Tasks

- [ ] Write the decision into `docs/design/fire-subsystem.md`
- [ ] Add a triage note to `CONTRIBUTING.md` so out-of-scope fire proposals get
      a consistent, non-dismissive response
- [ ] Open a tracking discussion for the eventual sibling repository

### Acceptance criteria

- Boundary documented and linked from the epic
- A concrete example of an in-scope and an out-of-scope proposal given in the
  docs

### References

- NWCG, NFDRS System Inputs and Outputs — https://www.nwcg.gov/publications/pms437/fire-danger/nfdrs-system-inputs-and-outputs

### PR DESCRIPTION
**Pull request by monocongo**
Closes #813.

## Summary

- Adds a `## Scope and boundary` section to `docs/design/fire-subsystem.md`: the in-scope criteria, the out-of-scope NFDRS/fuel/fire-behaviour set, the five sibling-repository criteria, and a concrete in-scope/out-of-scope example pair. The older one-paragraph exclusion in the Decision section folds into this section.
- Adds a `## Fire-weather scope` triage note to `CONTRIBUTING.md` pointing out-of-scope proposals at that boundary, so the response is consistent rather than dismissive.
- Opens the tracking discussion for the eventual sibling repository: https://github.com/monocongo/climate_indices/discussions/975.

## Verification

```
uv run ruff check src/ tests/
uv run ruff format --check src/ tests/
uv run mypy src/
uv run pytest
```

All green on the final head (2080 passed, 423 deselected). `main` was merged in after #973 landed. No `llms.txt` regeneration needed: neither changed file is in `SUMMARY_FILES` or `FULL_FILES`.

Noticed while verifying, left out to keep this PR scoped: `CONTEXT-MAP.md` still says "only Fosberg FFWI is implemented so far", which is stale now that KBDI, CFFWIS, HDW, and Haines have shipped.

## Summary by Sourcery

Document the fire-weather subsystem boundary and establish how out-of-scope proposals should be tracked for a potential sibling repository.

Enhancements:
- Define the fire-weather subsystem’s in-scope meteorological and climatological indices, out-of-scope operational fire-danger and fire-behavior modeling, and criteria for a future sibling repository.
- Add a consistent contributor triage path for fire-weather proposals that fall outside the subsystem boundary.

Documentation:
- Document the fire subsystem scope boundary, examples, sibling-repository criteria, and future proposal handling.

### CODE REVIEW

**Comment by monocongo** [comment]
### 🛠️ Multi-Agent Review Summary
- **Review target:** https://github.com/monocongo/climate_indices/pull/976
- **Lanes run:** 3 — correctness/edge cases, maintainability/conventions, tests/verification gates
- **Validation:** `uv run ruff check src/ tests/` pass; `uv run ruff format --check src/ tests/` pass; `uv run mypy src/` pass; `uv run pytest` pass (2080 passed, 423 deselected); independent anchor/scope re-check pass
- **Fix commits:** [177d3cd0 — docs(fire): fold the duplicated scope exclusions into the boundary section](https://github.com/monocongo/climate_indices/commit/177d3cd0), on top of the `origin/main` merge [a5827f67](https://github.com/monocongo/climate_indices/commit/a5827f67)

### Findings and fix status

#### [P3] Scope exclusions duplicated and asymmetric — Fixed
- **Location:** `docs/design/fire-subsystem.md` (Decision paragraph, formerly lines 41–43)
- **Origin:** Maintainability
- **Problem:** The Decision section restated the out-of-scope NFDRS/fuel/fire-behaviour set while the new canonical list omitted `ignition` and `occurrence prediction`, leaving two partial statements of the same boundary.
- **Applied fix:** Removed the duplicated paragraph and added `Ignition and occurrence prediction` to the out-of-scope list. Commit `177d3cd0`.

#### [P3] PR verification claim was on a stale base — Fixed
- **Location:** PR body and branch base (8 commits behind `main` after #973 merged)
- **Origin:** Tests
- **Problem:** The branch predated the #973 merge, so the PR body quoted a pre-merge test count that no longer matched the head CI logs.
- **Applied fix:** Merged `origin/main` (merge commit `a5827f67`), reran the full gate (2080 passed, 423 deselected), and updated the PR body.

#### [P3] Boundary not linked directly from the epic — Fixed
- **Location:** issue #793 body
- **Origin:** Tests
- **Problem:** #793 pointed at #813 but not at the recorded decision, so "boundary documented and linked from the epic" held only transitively.
- **Applied fix:** #793 now links `docs/design/fire-subsystem.md#scope-and-boundary` directly, and a tracking-discussion link was added (discussion #975).

#### [P3] `CONTEXT-MAP.md` stale implementation claim — Blocked
- **Location:** `CONTEXT-MAP.md:5` ("only Fosberg FFWI is implemented so far")
- **Origin:** Maintainability
- **Problem:** Pre-existing false statement; KBDI, CFFWIS, HDW, and Haines have all shipped. Unrelated to this diff.
- **Applied fix:** **Blocked:** out of scope for #813. Noted in the PR body for a separate fix.

#### [P4] No markdown link/anchor checker — Blocked
- **Location:** `tests/test_review_scripts.py` / repo-wide test conventions
- **Origin:** Tests
- **Problem:** A heading rename could silently break cross-document anchors; no existing test detects that, so the two new cross-links are verified only by review.
- **Applied fix:** **Blocked:** adding a repo-wide markdown-lint convention is out of scope for a documentation-decision ticket. Follow-up available if the maintainer wants one.

### Remaining blockers
None.

File: docs/design/fire-subsystem.md [hunk 1]
```diff
@@ -41,6 +41,45 @@ This family covers meteorological and climatological indices only. It excludes
...
+- Indices computable from standard meteorological and reanalysis fields
+- Deterministic, well-published algorithms with authoritative reference code
+- Anything that fits the existing NumPy + xarray + CF metadata + Dask pattern
+
+Out of scope:
+
+- NFDRS Ignition Component, Spread Component, Energy Release Component, and
+  Burning Index, which NWCG defines in terms of live and dead fuel moisture
+  and fuel models rather than weather alone
+- Fuel model catalogs, live and dead fuel state, and operational calibration
```
[the remark this answers is not in the document]
Comment (monocongo) [reply docs/design/fire-subsystem.md]: Fixed in 2e23e07b: the exclusion now names externally supplied fuel-model catalogs and live-fuel state as out of scope, and explicitly carves out the weather-driven CFFWIS moisture recursions (FFMC, DMC, DC) already shipped, so it no longer reads as contradicting that contract.

File: docs/design/fire-subsystem.md [hunk 2]
```diff
@@ -41,6 +41,45 @@ This family covers meteorological and climatological indices only. It excludes
...
+of these are true:
+
+1. An implementation requires fuel-model or fuel-state data structures
+2. An implementation requires operational calibration against fire-occurrence
+   records
+3. The fire code exceeds roughly 30 percent of the package's source volume
+4. Fire-specific dependencies would be forced on all `climate_indices` users
+5. The release cadence needs to diverge
+
+Until then, out-of-scope fire proposals are redirected to that sibling rather
```
[the remark this answers is not in the document]
Comment (monocongo) [reply docs/design/fire-subsystem.md]: Fixed in 2e23e07b: the triage guidance now routes an out-of-scope proposal to a fire-weather-labeled issue in this repo (an existing, usable destination) rather than the not-yet-created fire_weather_indices sibling.
